src/components/collapsible/Collapsible.js

```diff
@@ -117,7 +129,9 @@ class Collapsible extends React.PureComponent<Props, State> {
                         {title}
                         <IconCaretDown className="collapsible-card-header-caret" color={bdlGray50} width={8} />
                     </PlainButton>
-                    {isOpen && headerActionItems}
+                    {!!headerActionItems && (
+                        <span className="collapsible-card-header-actionItems">{headerActionItems}</span>
```

Comment:
I don't see CSS changes done in the PR? Wouldnt you want something like

```css
.collapsible-card {
   .collapsible-card-title + .collapsible-card-header-actionItems {
       display: none;
    }

  .collapsible-card-title:focus + .collapsible-card-header-actionItems,
   .collapsible-card-title:hover + .collapsible-card-header-actionItems {
       display: block;
    }

   &.is-open {
     .collapsible-card-title + .collapsible-card-header-actionItems {
         display: block;
      }
  }
}
```

Reply:
Also for new classes, i would switch to SUIT style

Reply:
I was under the assumption that these CSS changes can be done where ever its used. Can include it.

Reply:
Are you referring to the class `collapsible-card-header-actionItems`?